### ISSUE TITLE
App Store Upload Asset Validation Failed

### ISSUE THREAD

**Issue by msaravitz** [nativescript-community/ui-mapbox#115]
When attempting to upload to the App Store, the application receives an error of 
`Invalid Executable. The executable 'myapp.app/Frameworks/Mapbox.framework/Mapbox' contains bitcode.`

It appears bitcode is no longer supported by Apple and previous flags to disable it no longer function. (https://forums.developer.apple.com/forums/thread/764576)

Any help regarding this issue would be greatly appreciated, thanks!

MacOS 15.0
Xcode version 16.0
@nativescript-community/ui-mapbox  version 6.2.29

**Comment by RGBvision** [comment]
Same issue. https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.19.0 - Remove bitcode from framework binaries across the stack, thus unblocking AppStore submissions for client applications.

**Comment by farfromrefug** [comment]
@RGBvision @msaravitz Just published a new version based on Mapbox-IOS  6.4.1. I built it from source https://github.com/nativescript-community/mapbox-gl-native-ios. I built a xcframework and disabled bitcode so it should be good for appstore.
This is going to be my latest release on this repo. More work could be done to work on latest Mapbox 11.x but it would require a LOT of work. If you want it contact me and sponsor me for this.
